tools/util.wdl

```diff
@@ -278,74 +280,81 @@ task qc_summary {
 task compression_integrity {
     input {
         File bam
+        Int memory_gb = 4
+        Int modify_disk_size_gb = 0
         Int max_retries = 1
     }
 
     Float bam_size = size(bam, "GiB")
-    Int disk_size = ceil(bam_size + 10)
+    Int disk_size_gb = ceil(bam_size + 10) + modify_disk_size_gb
 
-    command {
+    command <<<
         bgzip -t ~{bam}
-    }
+    >>>
 
     output {
         String check = "passed"
     }
 
     runtime {
-        memory: "4 GB"
-        disk: disk_size + " GB"
+        memory: memory_gb + " GB"
+        disk: disk_size_gb + " GB"
         docker: 'quay.io/biocontainers/samtools:1.16.1--h6899075_1'
         maxRetries: max_retries
     }
 }
 
 task add_to_bam_header {
     input {
-        File input_bam
+        File bam
         String additional_header
-        String output_bam_name = basename(input_bam, ".bam") + ".reheader.bam"
+        String outfile_name = basename(bam, ".bam") + ".reheader.bam"
+        Int memory_gb = 4
+        Int modify_disk_size_gb = 0
         Int max_retries = 1
     }
 
-    Float bam_size = size(input_bam, "GiB")
-    Int disk_size = ceil(bam_size + 1)
+    Float bam_size = size(bam, "GiB")
+    Int disk_size_gb = ceil(bam_size + 10) + modify_disk_size_gb
 
     command <<<
-        samtools view -H ~{input_bam} > header.sam
+        samtools view -H ~{bam} > header.sam
         echo "~{additional_header}" >> header.sam
-        samtools reheader -P header.sam ~{input_bam} > ~{output_bam_name}
+        samtools reheader -P header.sam ~{bam} > ~{outfile_name}
     >>>
 
     output {
         File updated_header = "header.sam"
         Array[String] header_lines = read_lines("header.sam")
-        File reheadered_bam = output_bam_name
+        File reheadered_bam = outfile_name
     }
 
     runtime {
-        memory: "4 GB"
-        disk: disk_size + " GB"
+        memory: memory_gb + " GB"
+        disk: disk_size_gb + " GB"
         docker: 'quay.io/biocontainers/samtools:1.16.1--h6899075_1'
         maxRetries: max_retries
     }
 }
 
 task unpack_tarball {
+    # TODO I don't think I've ever checked if this task actually works.
+    #   I wrote it then realized there was a better way before this made it to testing
```

Comment:
This would also destroy any structure. If we keep the task, it should note that.